pkg/admin/brokerclient.go

```diff
@@ -407,13 +408,16 @@ func (c *BrokerAdminClient) GetUsers(
 		return nil, err
 	}
 
-	if err = util.DescribeUserScramCredentialsResponseResultsError(resp.Results); err != nil {
```

Comment:
I had added this error handler utility function before and discovered it had incorrect behavior